src/common/options.cc

```diff
@@ -5968,6 +5968,11 @@ std::vector<Option> get_rgw_options() {
     .set_min_max(1, 1024)
     .set_description("pg_num_min value for RGW metadata (omap-heavy) pools"),
 
+    Option("rgw_rados_pool_recovery_priority", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(5)
+    .set_min_max(-10, 10)
```

Comment:
This will help us prioritize recovery of the rgw pool over other pools in the same state by boosting `OSD_RECOVERY_PRIORITY_BASE`, but we might still be below the base priority of other states with higher base priority like `OSD_BACKFILL_INACTIVE_PRIORITY_BASE`.